src/ert/storage/local_ensemble.py

```diff
@@ -576,6 +576,10 @@ def load_parameters(
 
         return self._load_dataset(group, realizations)
 
+    def load_cross_correlations(self) -> xr.Dataset:
```

Comment:
This will fail if there is only Surface or Field parameters, or if no update has taken place, is the error understandable?

Reply:
Added this:

            raise FileNotFoundError(
                f"No cross-correlation data available at '{input_path}'. Make sure to run the update with "
                "Adaptive Localization enabled."
            )